test/rest.test.js

```diff
@@ -1592,22 +1586,23 @@ describe('strong-remoting-rest', function() {
           .end(expectErrorResponseContaining({ message: 'an error' }, done));
       });
 
-      it('should return 500 for unhandled errors thrown from before hooks', function(done) {
-        var method = givenSharedStaticMethod();
+      it('should return 500 for unhandled errors thrown from before hooks',
+        function(done) {
```

Comment:
To keep amount of white-space changes smaller, please use the following formatting:

```js
it('should return 500 for unhandled errors thrown from before hooks',
function(done) {
  // original code - no indentation change needed
});
```